packages/beacon-node/src/api/impl/validator/index.ts

```diff
@@ -556,20 +566,24 @@ export function getValidatorApi({
         // winston logger doesn't like bigint
         enginePayloadValue: `${enginePayloadValue}`,
         builderPayloadValue: `${builderPayloadValue}`,
+        consensusBlockValueEngine: `${consensusBlockValueEngine}`,
+        consensusBlockValueBuilder: `${consensusBlockValueBuilder}`,
```

Comment:
also log the total `blockValues..` so the user doesn't has to do math in his/her head